cdap-ui/app/directives/my-pipeline-runtime-args/my-pipeline-runtime-args.less

```diff
@@ -38,4 +38,13 @@ my-pipeline-runtime-args {
       }
     }
   }
+  .step-content-heading {
+    .step-content-subtitle {
+      font-weight: normal;
+    }
+  }
+  .runtime-arguments-values {
+    height: ~"calc(100% - 135px)";
```

Comment:
nit: maybe make this `140px`. Right now we're showing a little bit of the 5th runtime arg, and it looks like extra border. 
![image](https://user-images.githubusercontent.com/6516002/42792804-eca71d50-892b-11e8-84d4-818b513f7723.png)